generator/transit_generator.cpp

```diff
@@ -1,16 +1,135 @@
 #include "generator/transit_generator.hpp"
 
+#include "generator/osm_id.hpp"
+
+#include "routing_common/transit_serdes.hpp"
+#include "routing_common/transit_types.hpp"
+
+#include "coding/file_container.hpp"
+#include "coding/file_name_utils.hpp"
+#include "coding/file_writer.hpp"
+
+#include "platform/platform.hpp"
+
+#include "base/checked_cast.hpp"
 #include "base/logging.hpp"
 #include "base/macros.hpp"
 
+#include "3party/jansson/src/jansson.h"
+
+using namespace platform;
+using namespace routing;
+using namespace routing::transit;
+using namespace std;
+
+namespace
+{
+/// Drops all extensions from the filename.
```

Comment:
Extracts the file name from |filePath| and drops all extensions.

Reply:
Fixed